README.md

```diff
@@ -6,17 +6,21 @@ Run `npm install` then `npm start`, and visit `http://localhost:3000` for instru
 
 ### Introduction
 
-Hi! Thanks so much for interviewing with us 💖. If you are reading this, it means that we like you, and I hope you like us too! In this next step, you get an opportunity to see how we work, and we get the same opportunity for you. Let's get right to it.
+Hi! Thanks so much for interviewing with us 💖. If you are reading this, it means that we like you, and we hope you like us too! In this next step, you get an opportunity to see how we work, and we get the same opportunity for you. Let's get right to it.
 
 The task is to build a "people directory" page. This page should allow visitors to search by name for a person. If this were a real page, we'd want to search all 317 records in our CMS, but for the purposes of this exercise, limit your results to 100. This is the maximum number of results that our CMS API will return in a single page.
 
-Use [this Figma file](https://www.figma.com/file/xGicP4qkXbMhte4LAYxC4X/Untitled?node-id=0%3A1) as your source of truth for design and behavior:
+[Here's the "people directory" page! 👉](/people)
+
+_**Note:** The above link will only work when opened inside the project running locally._
+
+Use [this Figma file](https://www.figma.com/design/0c70HmKc0rkWCNdP9UYErO) as your source of truth for design and behavior:
 
 _**Note:** If you are unable to inspect elements in the design to get exact values for colors and font styles, make sure you're signed in with a Figma account, as we will expect your submission to match the design as closely as possible._
 
 There are definitely some parts of this that are a little ambiguous, which is how real world projects always work. Treat this the same way you would if you started working with us and this was your first day — this is quite similar to the type of task we would assign a new hire on their first day. We will set up a Discord channel for you to ask questions, get clarification, and collaborate with us in general, and we strongly encourage you to take advantage of this.
 
-We expect this to be done within a day or two's worth of full-time work - about 8 - 14 hours. We know that you likely have other commitments, and so we generally look at about week for candidates to complete the challenge. If you're approaching 14 hours of total time spent, just stop where you are and we'll complete the evaluation based on what you have done.
+We expect this to be done within 4 hours. We know that you likely have other commitments, and so we generally look at about week for candidates to complete the challenge. If you're approaching 4 hours of total time spent, just stop where you are, and we'll complete the evaluation based on what you have done. The code in your PR(s) should be the same quality as any code you would deploy to production.
```

Comment:
`The code in your PR(s) should be the same quality as any code you would deploy to production.` is redundant here. It's in the `"Submitting your work"` section already.

```suggestion
We expect this to be done within 4 hours. We know that you likely have other commitments, and so we generally look at about week for candidates to complete the challenge. If you're approaching 4 hours of total time spent, just stop where you are, and we'll complete the evaluation based on what you have done.
```